plugins/build.gradle.kts

```diff
@@ -4,26 +4,32 @@ plugins {
     id("com.gradle.plugin-publish") version "0.12.0" apply false
 }
 
+class FormaRootConfigurationException(
+    override val message: String,
+    override val cause: Throwable? = null
+): Exception()
+
+val propertyKotlinVersion = "forma.kotlinVersion"
+val propertyAgpVersion = "forma.agpVersion"
+
 val properties = java.util.Properties()
 val file = project.rootProject.file("../gradle.properties")
 try {
     file.inputStream().use { properties.load(it) }
 } catch (e: Throwable) {
-    throw Exception(
-        "Cannot read gradle.properties file from root project. " +
-                "Be sure that you have gradle.properties file in you root dir. " +
-                "Expected file path ${file.absolutePath}",
+    throw FormaRootConfigurationException(
+        "Can't read ${file.absolutePath}\nCreate file and declare $propertyKotlinVersion and $propertyAgpVersion",
         e
     )
 }
 
 // TODO: actually error will not be displayed, find the way to fix it
-fun errorNoProperty(propertyName: String): Nothing = error(
-    "Be sure that you have property with name $propertyName in you gradle.properties file, stored in ${file.absolutePath}"
+fun errorNoProperty(propertyName: String): Nothing = throw FormaRootConfigurationException(
+    "Can't find property $propertyName in ${file.absolutePath}"
 )
 
-val kotlinVersion = properties["forma.kotlinVersion"] ?: errorNoProperty("forma.kotlinVersion")
-val agpVersion = properties["forma.agpVersion"] ?: errorNoProperty("forma.agpVersion")
+val kotlinVersion = properties[propertyKotlinVersion] ?: errorNoProperty(propertyKotlinVersion)
```

Comment:
I see why you need Nothing here. 

```
properties[propertyKotlinVersion] ?: errorNoProperty(propertyKotlinVersion)
```

could be extracted as a separate method, doing that you'll not need `fun errorNoProperty`